metrics-util/src/layers/fanout.rs

```diff
@@ -100,6 +103,12 @@ pub struct Fanout {
     recorders: Vec<Box<dyn Recorder>>,
 }
 
+impl fmt::Debug for Fanout {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Fanout").finish_non_exhaustive()
```

Comment:
TIOLI: could always add a field like `recorders_len` with `self.recorders.len()` just to have _some_ meaningful information.

(Same below with `FanoutBuilder`.)

Reply:
Copilot suggested that too, but I removed it as I thought it seemed a little off. I was only 60/40 on removing it though, so given that you've also suggested it I'm inclined to put it back in.

Reply:
Done in c16f43b